### PR TITLE
Document editors changes

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/doc-editors/ACLEditor.vue [hunk 1]
```diff
@@ -329,4 +350,31 @@ export default defineComponent({
 :deep(.tag-input) {
   font-size: 0.58rem;
 }
+
+.collapsible {
```
Comment (Aviv-Galmidi) [inline src/doc-editors/ACLEditor.vue:354]: As these classes start repeating themselves often, let's remove their definitions from all the files and add them to `main.scss`

File: src/doc-editors/ACLEditor.vue [hunk 2]
```diff
@@ -2,69 +2,87 @@
...
-                          title="Custom Response">
-                    <option v-for="customResponse in customResponseNames"
-                            :value="customResponse[0]"
-                            :key="customResponse[0]">
-                      {{ customResponse[1] }}
-                    </option>
-                  </select>
+            <div class="columns">
+              <div class="column is-4">
+                <div class="content collapsible-content px-5 py-5">
```
Comment (Aviv-Galmidi) [inline src/doc-editors/ACLEditor.vue:21]: Maybe we should wrap the entirety of `columns` with this collapsible?

File: src/doc-editors/SecurityPoliciesEditor.vue [hunk 3]
```diff
@@ -1,70 +1,86 @@
...
-              </label>
-              <div class="control">
-                <textarea class="is-small textarea document-description"
-                          data-qa="description-input"
-                          title="Document description"
-                          v-model="localDoc.description"
-                          @input="emitDocUpdate"
-                          rows="5">
-                </textarea>
+          <div class="columns columns-divided">
```
Comment (Aviv-Galmidi) [inline src/doc-editors/SecurityPoliciesEditor.vue:18]: missing `collapsible-content`?

File: src/doc-editors/GlobalFiltersEditor.vue [hunk 4]
```diff
@@ -1,114 +1,131 @@
...
-            <div class="field">
-              <label class="label is-small">
-                Custom Response
-              </label>
-              <div class="control is-expanded">
-                <div class="select is-fullwidth is-small">
-                  <select v-model="localDoc.action"
+          <div class="columns">
+            <div class="column is-4">
+              <div class="content collapsible-content px-5 py-5">
```
Comment (Aviv-Galmidi) [inline src/doc-editors/GlobalFiltersEditor.vue:20]: Maybe we should wrap the entirety of `columns` with this collapsible?

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
Missing the following description number of rows changes:

- BackendServicesEditor
- ConfigTemplatesEditor
- DynamicRulesEditor
- EdgeFunctionsEditor
- MobileSDKsEditor
- RoutingProfilesEditor

File: src/doc-editors/SecurityPoliciesEditor.vue (hunk 3 above)
Comment (AvihayMReblaze) [reply src/doc-editors/SecurityPoliciesEditor.vue:18]: Done

File: src/doc-editors/ACLEditor.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/doc-editors/ACLEditor.vue:354]: Done

File: src/doc-editors/GlobalFiltersEditor.vue (hunk 4 above)
Comment (AvihayMReblaze) [reply src/doc-editors/GlobalFiltersEditor.vue:20]: Done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**